World.py

```diff
@@ -1158,7 +1158,7 @@ def bigocto_location(self) -> Optional[Location]:
         if self.cached_bigocto_location is not None:
             return self.cached_bigocto_location
         # Find an item location behind the Jabu boss door by searching regions breadth-first without going back into Jabu proper
-        if self.settings.logic_rules == 'glitched':
+        if False: #self.settings.logic_rules == 'glitched':
```

Comment:
Can you explain this?

Reply:
This conditional was originally added because glitched logic was not compatible with entrance rando and therefore did not have the Jabu boss door entrance defined. Now that it's compatible, it's no longer necessary.

I'm guessing the `if False` method was chosen to make this easier for Rob to maintain. Once these changes make their way into main Dev, the conditional should be removed completely.